sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -19,7 +19,7 @@
 
     <PackageReference Include="Azure.Core" />
     -->
-    <PackageReference Include="Azure.Core.Experimental" />
+    <!--    <PackageReference Include="Azure.Core.Experimental" />-->
```

Comment:
nit:  Would you mind updating the comment on L18 to reference Experimental rather than Core and `BinaryContent` instead of `the MessageWithMetatdata`?

_(I can't add a direct suggestion there)_